cfgov/preprocessed/css/molecules/image-and-text-25-75.less

```diff
@@ -0,0 +1,108 @@
+/* topdoc
+  name: Theme variables
+  family: cfgov-molecules
+  notes:
+    - "The following color and sizing variables are exposed,
+       allowing you to easily override them before compiling."
+  patterns:
+  - name: Colors
+    codenotes:
+      - |
+        @m-img_sm
+        @m-img_lg
+  tags:
+  - less
+*/
+@m-img_sm: 130px;
+@m-img_lg: 150px;
+
+
+/* topdoc
+  name: Image and Text 25/75
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+        <div class="m-image-and-text-25-75">
+            <div class="m-image-and-text-25-75_image-container">
+                <img src="image">
+            </div>
+            <div class="m-image-and-text-25-75_content">
+                <h1 class="m-image-and-text-25-75_heading">Heading</h1>
+                <p class="m-image-and-text-25-75_body">Body content</p>
+                <a class="m-image-and-text-25-75_link">Link</a>
+            </div>
+        </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .m-image-and-text-25-75
+            .m-image-and-text-25-75_image-container
+              img
+            .m-image-and-text-25-75_content
+              .m-image-and-text-25-75_heading
+              p.m-image-and-text-25-75_body
+              a.m-image-and-text-25-75_link
+  tags:
+    - cfgov-molecules
+*/
+
+.m-image-and-text-25-75 {
+    padding-top: unit( @grid_gutter-width / @base-font-size-px, em );
+    .u-clearfix();
+
+    &_image-container {
+
+        .respond-to-max( @mobile-max, {
+            text-align: center;
+            margin-bottom: unit( @grid_gutter-width / @base-font-size-px, em );
+        } );
+
+        .respond-to-min( @tablet-min, {
+            float: left;
+            margin-right: unit( @grid_gutter-width / @base-font-size-px, em );
+        } );
+    }
+
+    &_image {
+        width: @m-img_sm;
+        height: @m-img_sm;
+        background-size: contain;
+
+        .respond-to-min( @desktop-min, {
+            width: @m-img_lg;
+            height: @m-img_lg;
+        } );
+    }
+
+    &_content {
+        .respond-to-min( @tablet-min, {
+            margin-left: unit( ( @m-img_sm + @grid_gutter-width ) / @base-font-size-px, em );
+        } );
+
+        .respond-to-min( @desktop-min, {
+            margin-left: unit( ( @m-img_lg + @grid_gutter-width ) / @base-font-size-px, em );
+        } );
+    }
+
+    &_body,
+    &_link {
+        margin-bottom: unit( @grid_gutter-width / @base-font-size-px, em );
+    }
+
+    &_heading {
+        .h2(); /* Expose @font-size variable. */
+        margin-bottom: unit( @grid_gutter-width / @font-size, em );
+    }
+}
+
+h1.m-image-and-text-25-75_heading {
```

Comment:
I don't think so, he used the exposed `@font-size` in both instances and the spec actually says __not__ to use an `h1` because it'll compete with the page title.